includes/Modules/Search_Console/Email_Reporting/Report_Helper.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Search_Console\Email_Reporting\Report_Helper
+ *
+ * @package   Google\Site_Kit\Modules\Search_Console\Email_Reporting
+ * @copyright 2025 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Search_Console\Email_Reporting;
+
+/**
+ * Helper utilities for Search Console email reporting data processing.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Report_Helper {
```

Comment:
Do we need this class in this PR? I don't see it being mentioned in the issue.

If we need it, then let's probably come up with a better name for it to correctly express the purpose of this class. Something like `Report_Data_Processor`? 🤔